resources/scripts/install.sh

```diff
@@ -7,6 +7,19 @@ service docker restart
 
 certbot certonly --non-interactive --agree-tos --email ${certbot_email} --no-redirect --dns-route53 -d ${host_domain}
 
+SIZE=fdisk -l | grep nvme1n1 | awk '{ print $3 }'
+if [ "$SIZE" == "8" ]; then
```

Comment:
Here is a misleading thing, because you added the swap size like a module input, but its value is hardcoded here... Or this if condition should use the variable value. Or the swap size should be hardcoded and not able of being modified by module input

Reply:
Done, added the swap_volume_size variable.

Reply:
What if both volumes have the same size?